modules/nw-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+// * networking/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-load-balancer-operator_{context}"]
+= AWS Load Balancer Operator
+
+The AWS Load Balancer Operator can tag the public subnets if the `kubernetes.io/role/elb` tag is missing. Also, the AWS Load Balancer Operator detects the following from the underlying AWS cloud:
+
+* The ID of the virtual private cloud (VPC) on which the cluster hosting the operator is deployed in.
+* Public and private subnets of the discovered VPC.
+
+.Prerequisites
+
+* You must have the AWS credentials secret. The credentials are used to provide subnet tagging and VPC discovery.
+
+.Procedure
+
+. You can deploy the AWS Load Balancer Operator on demand from the OperatorHub, by creating a `Subscription` object:
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get sub aws-load-balancer-operator -o yaml | yq '.status.installplan.name'
+----
++
+.Example output
+[source,terminal]
+----
+install-zlfbt
+----
+
+. Check the status of an install plan. The status of an install plan must be `Complete`:
++
+[source,terminal]
+----
+oc -n aws-load-balancer-operator get ip <install_plan_name> -o yaml | yq .status.phase'
```

Comment:
I agree that the other places without a prompt are correct, but this one is an actual `oc` command that should have a prompt

Reply:
acked